### PR TITLE
Add options.

### PR DESCRIPTION
**Pull request by Varunvaruns9**
Adds various options for fixing errors in the REST API.

### CODE REVIEW

**Comment by rishi-s8** [comment]
@Varunvaruns9 can you remove the fake options at the top? Catches the eye.

**Comment by Varunvaruns9** [comment]
Which fake options?

File: tools-barebone/webservice/templates/upload.html [hunk 1]
```diff
@@ -1 +1,60 @@
...
+  </head>
+    <body>
+        <nav class="navbar navbar-inverse">
+            <div class="container-fluid">
+              <div class="navbar-header">
+                <a class="navbar-brand">CIF - Validator </a>
+              </div>
+              <ul class="nav navbar-nav">
+                <li class="active"><a href="/">Home</a></li>
+                <li><a href="upload">Upload file</a></li>
```
Comment (rishi-s8) [inline tools-barebone/webservice/templates/upload.html]: This should be removed.

File: tools-barebone/webservice/templates/upload.html [hunk 2]
```diff
@@ -1 +1,60 @@
...
+                <li><a href="view">View files</a></li>
```
Comment (rishi-s8) [inline tools-barebone/webservice/templates/upload.html]: This too.

File: tools-barebone/webservice/templates/upload.html (hunk 1 above)
Comment (rishi-s8) [reply tools-barebone/webservice/templates/upload.html]: The upload link.

File: tools-barebone/webservice/templates/upload.html (hunk 2 above)
Comment (rishi-s8) [reply tools-barebone/webservice/templates/upload.html]: The view link.

**Comment by rishi-s8** [comment]
Looks good. :)

**Pull request merged**